Backend/src/controllers/graph.controller.ts

```diff
@@ -0,0 +1,159 @@
+const { TelnetSocket } = require('telnet-stream');
+const net = require('net');
+import { Request, Response } from 'express';
+import { GRAPH_REMOVE_COMMAND, GRAPH_UPLOAD_COMMAND, LIST_COMMAND, TRIANGLE_COUNT_COMMAND } from './../constants/frontend.server.constants';
+import { ErrorCode, ErrorMsg } from '../constants/error.constants';
+
+let socket;
+let tSocket;
+
+const HOST = process.env.SERVER_HOST || '127.0.0.1';
+const PORT = parseInt(process.env.SERVER_PORT || '7776');
+const DEV_MODE = process.env.DEV_MODE === 'true';
+
+const connectToTelnet = (callback) => {
+  // If the global connection is undefined or closed, create a new connection
+  if (!socket || socket.destroyed) {
+    socket = net.createConnection(PORT, HOST, () => {
+      tSocket = new TelnetSocket(socket);
+
+      tSocket.on('do', (option) => {
+        tSocket.writeWont(option);
+      });
+
+      tSocket.on('will', (option) => {
+        tSocket.writeDont(option);
+      });
+
+      console.log('Telnet connection established');
```

Comment:
This log prints in to the Browser or on backend server log?

Reply:
on backend server

Reply:
Ok